metadata/service/remote_collector.yml

```diff
@@ -0,0 +1,7 @@
+applications:
+- heka
+_param:
+parameters:
+  heka:
+    remote_collector:
+      enabled: true
```

Comment:
yes, this is WIP, we must decide on the definitions first, so there's no rework.